packages/venia-concept/src/ServiceWorker/Utilities/__tests__/imageCacheHandler.spec.js

```diff
@@ -96,6 +96,25 @@ describe('Testing findSameOrLargerImage', () => {
         expect(returnedResponse.url).toBe(expectedUrl);
     });
 
+    test('Should not return response if URL for same filename is not available', async () => {
+        const fileName1 = 'fileName1';
+        const fileName2 = 'fileName2';
```

Comment:
Since most images had the same extension, this more more about the contains matching that end. I think this covers it more accurately:
```suggestion
        const fileName1 = 'prefixed-file-name.jpg';
        const fileName2 = 'file-name.jpg';
```

`file-name.jpg` would contains match against `https://develop.pwa-venia.com/media/catalog/p/r/prefixed-file-name.jpg?auto=webp&format=pjpg&width=1600&height=2000`

Reply:
You are right, thanks for the suggestion dude.